.dotnet/src/Generated/Models/StreamingChatCompletionUpdate.Serialization.cs

```diff
@@ -34,6 +34,11 @@ void IJsonModel<StreamingChatCompletionUpdate>.Write(Utf8JsonWriter writer, Mode
             writer.WriteNumberValue(CreatedAt, "U");
             writer.WritePropertyName("model"u8);
             writer.WriteStringValue(Model);
+            if (Optional.IsDefined(ServiceTier))
+            {
+                writer.WritePropertyName("service_tier"u8);
+                writer.WriteStringValue(ServiceTier);
+            }
```

Comment:
This one is also nullable. Just double-checking the serialization in the Write and Deserialize methods.